python/lib/sift_py/ingestion/rule/config.py

```diff
@@ -166,10 +176,17 @@ def interpolate_namespace_rule(
                     action = RuleActionCreateDataReviewAnnotation(
                         assignee=rule.get("assignee"), tags=tags
                     )
-                return description, expression, rule_client_key, action, asset_names, tag_names
+                return (
+                    description,
+                    cast(str, expression),
```

Comment:
@niliayu this is one of the areas that we missed. I'm casting this to a string to get this to work but if you remove the cast `pyright` will tell you what's wrong.